src/Handlers/GetProjectsHandler.ts

```diff
@@ -0,0 +1,16 @@
+import { bind, Handler } from '../Bus';
+import { GetProjects } from './Events';
+import { JiraClient, JiraProject } from '../Integrations';
+import { inject, injectable } from 'tsyringe';
+
+@injectable()
+@bind(GetProjects)
+export class GetProjectsHandler implements Handler<GetProjects, JiraProject[]> {
+  constructor(
+    @inject(JiraClient) private readonly jiraClient: JiraClient
+  ) {}
+
+  public async handle(_event: GetProjects): Promise<JiraProject[]> {
```

Comment:
`async` statement is redundant
```suggestion
  public handle(_event: GetProjects): Promise<JiraProject[]> {
```